l10n_nl_xaf_auditfile_export/data/XmlAuditfileFinancieel3.2.xsd

```diff
@@ -862,7 +869,6 @@
 			<xsd:enumeration value="TZ"/>
 			<xsd:enumeration value="UA"/>
 			<xsd:enumeration value="UG"/>
-			<xsd:enumeration value="UK"/>

```

Comment:
Shouldn't deleted values be kept instead? For backwards compatibility, to avoid failures while exporting old records?

Reply:
To be honest, I don't think so. I don't think there never was a country with an ISO UK code (although it's common to confuse it with GB of course). And in other cases, it's countries that are renamed. The old codes are not valid anymore. It seems entirely possible that the tax authority won't accept these illegal country codes, so it's important to keep up to date. The change in Odoo is limited to changing the country code of one or two countries.

Reply:
Yes, I see.
I just checked: `AN` code is still [present in Odoo 10.0](https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/res/res_country_data.xml#L56) although it was removed in 11.0+ with https://github.com/odoo/odoo/commit/bfbaaa7d1319d109018b4e77ff930dadb8211cee#diff-c935af95c0540fa947cf91a105585dd4